Add email making competition virtual (#51)
srobo/team-emails: SR2020/2020-05-22-virtual-competition-transition.md

---
to: sr2020-teams

> subject: Transitioning Student Robotics 2020 into a virtual competition

Reviewer: I'm concerned at the length of the subject here, might be cut off.

As the sender will show as us, I'd be tempted to redact that
```suggestion
subject: Transitioning to a virtual competition
```

---

Hi all,

> In light of everything happening, and the uncertainty it brings, we're unfortunately cancelling the in-person competition for Student Robotics 2020.

Reviewer: ```suggestion
In light of recent events, and the uncertainty it has brought, we're unfortunately cancelling the in-person competition for Student Robotics 2020.
```

## Virtual Competition

> The virtual competition event will take place over a few weekends in July, with a few weekends of league matches, and a knockout final.

Reviewer 2: Perhaps "sessions" rather than "weekends"?
At the moment this reads like the whole weekend might be filed with matches, but I don't think that's what it's trying to say.

> As we said in the previous email, we aim to keep the virtual competition rules as similar to Two Colours as possible so teams can use the code they have been working on since Kickstart.

Reviewer 2: This is still talking about re-using code, yet with the mode to Python 3 and the other (subtle but still present) changes to the API it feels slightly misleading to say that.

Is there something we can say which is a bit more general? Perhaps just say something like "easily build on the work they have done since Kickstart"?

The [forum](https://studentrobotics.org/forum/) will be open and staffed should you need help from us blueshirts.

> We will be announcing the final details of the virtual competition early next week.

Reviewer: ```suggestion
We will be announcing the full details of the virtual competition early next week.
```
